src/data/blog/web-developer-earn-the-salary-you-deserve.us.md

```diff
@@ -26,12 +27,16 @@ We all need to move around on the web and we need to do it in the most adequate
 <tbody >
 <tr >
 
-<td colspan="2" >**Quick Facts: Web Developers**
+<td colspan="2" >
+
+**Quick Facts: Web Developers**
```

Comment:
added white space because it is poorly structured (it becomes text instead of a link)